app/controllers/application_controller.rb

```diff
@@ -61,13 +62,15 @@ def deny_access
     (User.current.logged? || request.xhr?) ? render_403 : require_login
   end
 
+  def require_ssl?
+    SETTINGS[:require_ssl]
+  end
+
   def require_ssl
```

Comment:
It is used in https://github.com/theforeman/foreman/pull/2328/files#diff-55c5b7aecfb519d0e4880eaf2788eb6eR4
which is like a before_filter